cmd/kubeadm/app/phases/controlplane/manifests.go

```diff
@@ -228,6 +229,48 @@ func getAPIServerCommand(cfg *kubeadmapi.MasterConfiguration, k8sVersion *versio
 	return command
 }
 
+// calcNodeCidrSize determines the size of the subnets used on each node, based
+// on the pod subnet provided.  For IPv4, we assume that the pod subnet will
+// be /16 and use /24. For IPv6, there are some restrictions. First, the pod
+// subnet size must be /66 or larger. Second, the bit mask used for allocating
+// subnets (one per node) allows up to 32K (16 bits). To accomodate this, the
+// following algorithm is used, based on pod subnet size (implying the number of
+// bits available), to determine the node CIDR size.
+//
+// POD SUBNET SIZE   BITS AVAIL   ACTION
+//    /66  to /104   62 to 24     Allow 16 bits for node CIDR, rest for pods/node
+//    /105 to /112   16 to 23     Give node CIDR half of the bits available
+//    /113 to /119   15 to  9     Give 8 bits for pods/node and rest for node CIDR
```

Comment:
Note: With /119, we can only have one node, with 256 pods. Should that range be changed to /113 - /118, and consider /119 an invalid configuration, or reduce the number of pods/node to 128 for /119?

Reply:
Correction... with /119 we could have two nodes, with 256 pods. Would that limitation be OK, or would we want to require /118 as the limit or restrict the number of pods/node to 128 in that case?